src/blocks.h

```diff
@@ -33,10 +33,10 @@ namespace libsemigroups {
   //! Class for signed partitions of the set \f$\{0, \ldots, n - 1\}\f$
   //!
   //! It is possible to associate to every Bipartition a pair of blocks,
-  //! Bipartition::left_blocks and Bipartition::right_blocks, which determine
-  //! the Green's \f$\mathscr{L}\f$- and \f$\mathscr{R}\f$-class of the
-  //! Bipartition in the monoid of all partitions. This is the purpose of this
-  //! class.
+  //! Bipartition::left_blocks and Bipartition::right_blocks, which (together
+  //! with information on which blocks are transverse) determine the Green's
```

Comment:
The a `Blocks` object contains information about which blocks are transverse, so this is not completely accurate.